src/screens/WasteCollectionScreen.js

```diff
@@ -17,14 +17,15 @@ import {
   Wrapper,
   WrapperWithOrientation
 } from '../components';
-import { colors, device, normalize, texts } from '../config';
+import { colors, device, namespace, normalize, secrets, staticRestSuffix, texts } from '../config';
 import { useQuery } from 'react-apollo';
 import { getQuery, QUERY_TYPES } from '../queries';
 import { useRefreshTime } from '../hooks';
-import { graphqlFetchPolicy, setupLocales } from '../helpers';
+import { graphqlFetchPolicy, openLink, setupLocales } from '../helpers';
 import { NetworkContext } from '../NetworkProvider';
 import { getInAppPermission, showPermissionRequiredAlert } from '../pushNotifications';
 import { renderArrow } from '../components';
+import WebView from 'react-native-webview';
```

Comment:
could be organized to the top section of external imports

Reply:
Webview is now removed anyways :)